### DISCUSSION TITLE
Is cppfront considering to stick around?

### DISCUSSION THREAD

**Discussion by Parad0x84** [hsutter/cppfront#1086] [Q&A] [answered]
Is cppfront considering to stick around for long term just like typescript or is it plannig to be deprecated at some point? (if it succeeds)
Also I think this is a question that probably asked a lot, but I couldn't find it anywhere...

**Comment by hsutter** [comment, marked as answer]
Thanks for the interest! It's currently a personal experiment, but so far it's continuing. See also #1084.
